app/src/main/java/com/monkeyteam/chimpagne/model/location/Nominatim.kt

```diff
@@ -16,6 +17,10 @@ object NominatimConstants {
   var PORT = 443
 }
 
+fun getCurrentLanguage(): String {
```

Comment:
This could be useful outside of Nominatim.
I suggest putting it in another file.
However, one could argue that for now it is only used here, so to me it's not that big of a deal.

Reply:
Dedicated LanguageTools.kt created. Thank you